public/css/README

```diff
@@ -1,2 +0,0 @@
-Don't modify these files.
-They are generated from SCSS code in ui/ modules.
```

Comment:
We can probably leave this file in place, it should only help newcomers to the code base.

Reply:
This was a mistake

Reply:
Fixed this in recent commit